File: Python course/Proyects/Pretty price/pretty_price.py
#pretty_price(3.21, 99) -> 3.99
#pretty_price(3.21, 0.99) -> 3.99 

def pretty_price(price, decimal):
    if isinstance(decimal, int):
        decimal /= 100
        
    return int(price) + decimal
      

print(pretty_price(3.21, 0.98))
print(pretty_price(5.57, 95))

File: JavaScript/Modern Tools/Ejercicios/prettyPrice.js
function prettyPrice(price, decimal) {
    if (Number.isInteger(decimal)) {
        decimal *= 0.01;
    }

    return Math.floor(price) + decimal;
}

console.log(prettyPrice(5.2, 87));
console.log(prettyPrice(5.2, 0.87));